src/domains/course/services/courseCreateService.ts

```diff
@@ -155,3 +144,73 @@ export const getCategories = async (): Promise<Category[]> => {
     throw new Error('카테고리 목록을 불러오는 중 오류가 발생했습니다.');
   }
 };
+
+// 강좌 수정 API
+export const updateDraftCourse = async (
+  courseId: string,
+  payload: Partial<CourseDraftForm>,
+): Promise<void> => {
+  if (!courseId) throw new Error('Invalid courseId');
+
+  try {
+    const formData = createCourseFormData(payload);
+
+    await patchApi<void>(`/api/instructor/courses/${courseId}`, undefined, {
+      body: formData,
+      credentials: 'include',
+    });
+
+    // 보통 수정 API는 body가 없거나 {status, code, message} 정도만 반환하므로 따로 파싱 안 해도 됨
+  } catch (err) {
+    console.error('updateDraftCourse 실패:', err);
+    throw err instanceof Error ? err : new Error('강좌 수정 중 오류가 발생했습니다.');
+  }
+};
+
+// 생성된 강좌조회 용 데이터 매핑 함수
+const mapResponseToCourseDraft = (data: any) => {
+  //카테고리
+  const category = Array.isArray(data.categoryIds)
+    ? data.categoryIds.map((id: unknown) => String(id))
+    : data.category
+      ? [String(data.category)]
+      : [];
+  // 강좌 정보 가공
+  const courseDraft: CourseDraftForm = {
+    title: data.title ?? '',
+    summary: data.summary ?? '',
+    description: data.description ?? '',
+    thumbnail: data.thumbnailUrl ?? '',
+    category,
+    level: data.courseLevel ?? '',
+    price: data.price ?? 0,
+    status: data.status ?? 'draft',
+  };
+  // 섹션 및 강의 정보 가공
+  const sectionDrafts: SectionDraftForm[] = (data.sections ?? []).map((sec: any) => ({
+    id: String(sec.id),
+    title: sec.title ?? '',
+    status: sec.status ?? 'draft',
+    lectures: (sec.lectures ?? []).map((lec: any) => ({
+      id: String(lec.id),
+      title: lec.title ?? '',
+      duration: lec.totalDurationSeconds ?? 0,
+      videoUrl: lec.videoUrl ?? '',
+      resource: lec.resource,
+    })),
+  }));
+
+  return { courseDraft, sectionDrafts };
+};
+
+// 임시 생성된 강좌 조회API
+export async function fetchCourseWithSections(courseId: string): Promise<{
```

Comment:
```suggestion
export async function getDraftCourse(courseId: string): Promise<{
```
함수명 컨벤션에 맞게 직관적으로 변경하는거 어떠신가요?!

Reply:
getDraftCourse 로 이름 전체 변경했습니다~